drivers/clk/renesas/rzg2l-cpg.c

```diff
@@ -158,6 +164,44 @@ struct rzg2l_cpg_priv {
 	struct rzg2l_pll5_mux_dsi_div_param mux_dsi_div_params;
 };
 
+/**
```

Comment:
@long-trinh-rvc,

We don't need this function to check dsi status from device tree. Please move it.